src/plugins/intel_cpu/src/node.cpp

```diff
@@ -565,6 +566,7 @@ void Node::updateDynamicParams() {
     IE_ASSERT(isDynamicNode()) << "Node::updateDynamicParams() is called to a static shape node of type: " << getTypeStr() << " with name: " << getName();
     if (isExecutable()) {
         if (needPrepareParams()) {
+            PERF_PREPARE_PARAMS(this);
```

Comment:
It may be useful to include `needPrepareParams` call into prepareParams time measurements.